examples/individual_properties/main.rs

```diff
@@ -37,7 +38,10 @@ struct Config {
 }
 
 // Getter function for the property "name" of a sensor.
-async fn get_name_for_sensor<T: PropAccess>(device: &T, sensor_n: i32) -> Result<String, DynError> {
+async fn get_name_for_sensor(
+    device: &impl PropAccess,
```

Comment:
The reference before the impl should not be needed

Reply:
This goes for other impls in the examples

Reply:
I've tried it and it does not work, this is the error i receive:
```
the trait `astarte_device_sdk::prelude::PropAccess` is not implemented for `&astarte_device_sdk::AstarteDeviceSdk<astarte_device_sdk::store::SqliteStore, astarte_device_sdk::transport::mqtt::Mqtt>`
```
And i cannot move the value because the function gets called two times in sequence, and that would be a use of a moved value.

Reply:
Never mind then TIL 👍🏻